app/screens/channel/channel.ios.js

```diff
@@ -102,6 +93,15 @@ export default class ChannelIOS extends ChannelBase {
                         valueEvent={CHANNEL_POST_TEXTBOX_VALUE_CHANGE}
                     />
                 }
+                <View nativeID={ACCESSORIES_CONTAINER_NATIVE_ID}>
```

Comment:
Why move it?

Reply:
Because otherwise the autocomplete rendered behind the post draft, even when setting `elevation: 1` on the autocomplete (the same was true on Android, which is why I had to move the Autocomplete on there as well)